draft-ietf-quic-transport.md

```diff
@@ -994,6 +994,22 @@ packets sent from only one local address.  An endpoint that migrates away from a
 local address SHOULD retire all connection IDs used on that address once it no
 longer plans to use that address.
 
+An endpoint can request that its peer retire connection IDs by sending a
+NEW_CONNECTION_ID frame with an increased Retire Prior To field.  Upon receipt,
+the peer SHOULD retire the corresponding connection IDs and send the
+corresponding RETIRE_CONNECTION_ID frame in a timely manner.  Failing to do so
+can cause packets to be delayed or lost and harm connection performance as the
```

Comment:
I'm a bit vague on "delayed or lost and harm connection performance as the original endpoint might not route those connection IDs optimally after some delay."  Instead, how about "delayed, lost, or cause the original endpoint to send a stateless reset in response to a connection ID it can no longer route correctly."?